Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM adoptopenjdk/openjdk11
 RUN mkdir /app
-COPY target/xyz-tile-cache-0.0.1-SNAPSHOT.jar /app/xyz-tile-cache.jar
+COPY target/xyz-tile-cache-0.3.0.jar /app/xyz-tile-cache.jar
```

Comment:
Could you change this to 0.4.0 here and in the pom? For this reapo I have been updating the version with every change since it doesn't change that often.